packages/editor/src/plugins/text/components/suggestion-item.tsx

```diff
@@ -0,0 +1,70 @@
+import IconFallback from '@editor/editor-ui/assets/plugin-icons/icon-fallback.svg'
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import React, { forwardRef } from 'react'
+
+import { SuggestionOption } from '../hooks/use-suggestions'
+import { cn } from '@/helper/cn'
+
+interface SuggestionItemProps {
+  option: SuggestionOption
+  selected: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onFocus: () => void
+  onMouseEnter: () => void
+  onMouseLeave: () => void
+  onBlur: () => void
+  tooltipPosition?: 'right' | 'left'
+}
+
+export const SuggestionItem = forwardRef<
+  HTMLButtonElement,
+  SuggestionItemProps
+>((props, ref) => {
```

Comment:
```suggestion
>(function SuggestionItem(props, ref) {
```

And then you don't need to manually set displayName at the bottom of the file.

Reply:
Thanks, Fixed :heavy_check_mark: